test/images/agnhost/netexec/netexec.go

```diff
@@ -255,6 +258,11 @@ func clientIPHandler(w http.ResponseWriter, r *http.Request) {
 	log.Printf("GET /clientip")
 	fmt.Fprintf(w, r.RemoteAddr)
 }
+func headerHandler(w http.ResponseWriter, r *http.Request) {
+	key := r.FormValue("key")
```

Comment:
if key is not present, does it dumps all headers?

Reply:
It does not, it will be an empty string. If we wanted to dump all headers we could allow a client to possibility to target `/headers` and return a whitespace separated slice of all header values.

Reply:
I think we should have only one handler that by default return all headers, with the option of specifying a key.

Reply:
I agree, this is what I ended up doing just now.